src/Services/Air/Air.js

```diff
@@ -296,6 +296,15 @@ module.exports = (settings) => {
               if (allTicketsVoid) {
                 return Promise.resolve(true);
               }
+              // Check for REFUNDED
+              const allTicketsRefunded = ticketData.tickets.every(
```

Comment:
There is no need in checking array twice. All tickets should be either VOID or REFUND